environments/values.yaml

```diff
@@ -240,6 +240,9 @@ applications:
   # -- Enable the vo-cutouts application
   vo-cutouts: false
 
+# -- Add labels for an application namespace.
+namespaceLabels: {}
```

Comment:
The placement of this setting implies that it is being added to every namespace, but it's only added to sasquatch. Is the intent to apply it to other namespaces later and this is just for testing? If we're going to eventually need to pick which namespaces it's applied to and which it isn't, I think this needs different structure (perhaps a mapping of namespace names to labels).

Reply:
It was supposed to be generic, but I see your point. If another app uses the config parameter, they'll pick up labels they probably don't want. Let me think about how this could be done with you mapping suggestion.

Reply:
OK, I went a different route by using applications to hide the labelling from others that don't care. Let me know if this is going in the right direction.